extension/src/plots/model/custom.ts

```diff
@@ -23,3 +25,20 @@ export const doesCustomPlotAlreadyExist = (
   order.some(value => {
     return value.param === param && value.metric === metric
   })
+
+const removeColumnTypeFromPath = (columnPath: string, type: string) =>
+  columnPath.startsWith(type + FILE_SEPARATOR)
```

Comment:
@julieg18 if you import anything from `../../experiments/columns/paths` here you expose the webview to 'path' so it won't build.

Reply:
because you import `isCheckpointPlot` into the webview